plugins/org.jkiss.dbeaver.ext.bigquery/src/org/jkiss/dbeaver/ext/bigquery/model/BigQueryDataSource.java

```diff
@@ -53,7 +52,7 @@ protected String getConnectionURL(DBPConnectionConfiguration connectionInfo) {
             DBPDriver driver = getContainer().getDriver();
             return driver.getDataSourceProvider().getConnectionURL(driver, connectionInfo);
         }
-        return connectionURL;
+        return connectionURL + ";EnableSession=1;";
```

Comment:
It cannot be anbled by default. We have this feature in desktop, add parameter in config